templates/repo/issue/new_form.tmpl

```diff
@@ -136,6 +136,64 @@
 				</div>
 			</div>
 
+			{{if .IsProjectsEnabled}}
+			<div class="ui divider"></div>
+
+			<input id="project_id" name="project_id" type="hidden" value="{{.project_id}}">
+			<div class="ui {{if not .HasIssuesOrPullsWritePermission}}disabled{{end}} floating jump select-project dropdown">
+			  <span class="text">
+			    <strong>{{.i18n.Tr "repo.issues.new.projects"}}</strong>
+			    {{if .HasIssuesOrPullsWritePermission}}
+			      {{svg "octicon-gear" 16}}
+			    {{end}}
+			  </span>
+			  <div class="menu">
+			    <div class="header" style="text-transform: none;font-size:16px;">{{.i18n.Tr "repo.issues.new.add_project_title"}}</div>
+			    {{if or .OpenProjects .ClosedProjects}}
+			    <div class="ui icon search input">
+			      <i class="search icon"></i>
+			      <input type="text" placeholder="{{.i18n.Tr "repo.issues.filter_projects"}}">
```

Comment:
This needs localization cause it doesn't display properly. I can add the English one but, for the rest should add them.

Reply:
@TsakiDev souch things just add an english translation into `options/locale/locale_en-US.ini` the rest crowdin will take care of :)

and yes would be nice if you add it :)

Reply:
Done! 😉 adelowo#13